pkg/ride/functions_list.go

```diff
@@ -4,10 +4,14 @@ import (
 	"sort"
 
 	"github.com/pkg/errors"
+
 	"github.com/wavesplatform/gowaves/pkg/ride/math"
 )
 
-const maxListSize = 1000
+const (
+	maxListSize = 1000
+	threeArgs   = 3
```

Comment:
I think `threeArgs` constant should be inside the function body where it is used.